lib/time/exceptions.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Timex.Time.InvalidUnitError do
```

Comment:
Looks good, but can you change the spacing to 2 spaces per tab instead of 4? Looks like only this file has tab-width of 4 for some reason.